mmpose/datasets/pipelines/mesh_transform.py

```diff
@@ -0,0 +1,370 @@
+import cv2
+import mmcv
+import numpy as np
+from torchvision.transforms import functional as F
+
+from mmpose.core.post_processing import (affine_transform, fliplr_joints,
+                                         get_affine_transform)
+from mmpose.datasets.registry import PIPELINES
+
+
+def flip_smpl_pose(pose):
+    """Flip SMPL pose parameters horizontally.
+
+    Args:
+        pose (np.ndarray([72])): SMPL pose parameters
+
+    Returns:
+        pose_flipped
+    """
+
+    flippedParts = [
+        0, 1, 2, 6, 7, 8, 3, 4, 5, 9, 10, 11, 15, 16, 17, 12, 13, 14, 18, 19,
```

Comment:
notice the change of order here `6, 7, 8` and `3, 4, 5`.
is it intended?

Reply:
okay i guess it is part of the flip

Reply:
Yes, it's used to filp the SMPL pose parameters.